test/tests/functional/pbs_alps_inventory_check_hook.py

```diff
@@ -131,7 +136,8 @@ def test_cray_login_nodes(self):
         """
         Test log when no nodes with vntype 'cray_login' are present.
         """
-        now = time.time()
+        mc = ""

```

Comment:
@bhroam Regarding self.mom, Indeed it is false positive, because if you are in test case then self.mom will never be uninitialized... If PBSTestSuite can't initialize self.mom then it will mark setUp as failed and test case will never execute...
May be this false positive is because in next line we have self.mom = self.moms.values()[0] so it codacy thinks that test case is creating self.mom while in real execution PBSTestSuite creates self.mom variable...

saying that @riyazhakki You don't need 'self.mom = self.moms.values()[0]' because PBSTestSuite does that part for you... see [here](https://github.com/PBSPro/pbspro/blob/master/test/fw/ptl/utils/pbs_testsuite.py#L690)